typing/ctype.ml

```diff
@@ -3519,7 +3520,7 @@ let match_class_types ?(trace=true) env pat_sch subj_sch =
              | _      -> CM_Hide_public lab::err
              end
            in
-           if Concr.mem lab sign1.csig_concr then err
+           if lab = dummy_method || Concr.mem lab sign1.csig_concr then err
```

Comment:
Why this change? We are comparing class types, so they are bound to both contain the dummy method, no?